static/app/views/replays/detail/console/index.tsx

```diff
@@ -1,19 +1,26 @@
 import styled from '@emotion/styled';
+import moment from 'moment';
 
 import {Panel} from 'sentry/components/panels';
 import {BreadcrumbTypeDefault} from 'sentry/types/breadcrumbs';
+import {getFormattedDate} from 'sentry/utils/dates';
 
 import ConsoleMessage from './consoleMessage';
 
 interface Props {
   breadcrumbs: BreadcrumbTypeDefault[];
+  startTimestamp: number;
 }
 
-function Console({breadcrumbs}: Props) {
+function Console({breadcrumbs, startTimestamp = 0}: Props) {
   return (
     <ConsoleTable>
       {breadcrumbs.map((breadcrumb, i) => (
         <ConsoleMessage
+          relativeTimestamp={getFormattedDate(
+            (moment(breadcrumb.timestamp).unix() - startTimestamp) * 1000,
+            'HH:mm:ss'
```

Comment:
there are some new helpers we've got for these date/time conversions: https://github.com/getsentry/sentry/blob/master/static/app/components/replays/utils.tsx#L22-L30

Reply:
Oh man I love this, thanks for the heads up!